app/config/config.yml

```
@@ -24,6 +24,7 @@ parameters:
     pipeprint_url: '%env(ENVELOPER_PIPEPRINT_URL)%'
     env(ENVELOPER_PIPEPRINT_URL): ~
     env(ENVELOPER_QUEUE_DSN): 'spool://memory'
+    env(DEFAULT_TIMEZONE): 'Europe/London'
```

Since we have to provide some sort of default, I'd think `UTC` is the least-surprising to the largest amount of people. 

Would you agree?

```suggestion
    env(DEFAULT_TIMEZONE): 'UTC'
```